src/main/java/org/dotbase/providers/TOTPProvider.java

```diff
@@ -101,17 +100,12 @@ private UserModel isValidUser(RealmModel realm, String username) {
     return user;
   }
 
-  private UserCredentialStore getCredentialStore() {
-    return session.userCredentialManager();
-  }
-
   public boolean configuredOTPCredential(RealmModel realm, UserModel user) {
-    if (getCredentialStore().getStoredCredentialsByTypeStream(realm, user, "totp").count() > 0) return true;
-    return getCredentialStore().getStoredCredentialsByTypeStream(realm, user, "otp").count() > 0;
+    return user.credentialManager().getStoredCredentialsByTypeStream("totp").count() > 0;
```

Comment:
```suggestion
   if(user.credentialManager().getStoredCredentialsByTypeStream("totp").count() > 0) return true;
   return user.credentialManager().getStoredCredentialsByTypeStream("otp").count() > 0;
```

Ich habe bei der Implementierung glaube ich was übersehen.